manifests/01-rbac.yaml

```diff
@@ -134,6 +134,23 @@ rules:
   - machine-approver
   verbs:
   - update
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - networks
+  verbs:
+  - get
+  - list
```

Comment:
why do we need any other than get?

Reply:
shouldn't this be using a regular client rather than a cache for this?

Reply:
https://github.com/openshift/cluster-machine-approver/blob/7b7289a9e6419b9a5ce60cba22b60b9d8ddf3ed5/main.go#L155-L158

Reply:
We could add this as uncached and then avoid the informer, but I don't know if we particularly need to.

In particular, we expect the network object to never change during the lifetime of the cluster and the hostsubnets should be pretty static as well